src/shared/clients/ecsClient.ts

```diff
@@ -95,4 +95,23 @@ export class DefaultEcsClient {
     protected async createSdkClient(): Promise<ECS> {
         return await ext.sdkClientBuilder.createAwsService(ECS, undefined, this.regionCode)
     }
+
+    public async executeCommand(
+        cluster: string,
+        container: string,
+        task: string,
+        command: string
+    ): Promise<ECS.ExecuteCommandResponse> {
+        const sdkClient = await this.createSdkClient()
+
+        const params: ECS.ExecuteCommandRequest = {
+            command: command,
+            interactive: true, // Required
```

Comment:
Please link to a reference that explains why "true" is required here.

Reply:
I'm not sure there's a specific link - but ECS Exec doesn't support non-interactive mode right now, according to this blog support is planned in the "near future": https://aws.amazon.com/blogs/containers/new-using-amazon-ecs-exec-access-your-containers-fargate-ec2/

Reply:
The docs don't go into detail why, but it's stated that the flag is required and needs a `true` value.  https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ExecuteCommand.html

Reply:
Both of those links are worth mentioning here :)

Reply:
Both have been added as line comments.